ScoutSuite/providers/aws/resources/ec2/ami.py

```diff
@@ -16,9 +20,5 @@ async def fetch_all(self):
     def _parse_image(self, raw_image):
         raw_image['id'] = raw_image.get('ImageId')
         raw_image['name'] = raw_image.get('Name')
-        raw_image['arn'] = 'arn:aws:ec2:{}:{}:ami/{}'.format(self.region,
-                                                            raw_image.get('OwnerId'),
-                                                            raw_image.get('ImageId'))
-        if 'Tags' in raw_image:
-            raw_image['tags'] = {x["Key"]: x["Value"] for x in raw_image["Tags"]}
-            return raw_image['id'], raw_image
```

Comment:
Might have to re-add this.

Reply:
We had forked from their branch because they didn't (yet) have tagging included, and it appears that this merge will remove that addition.  It also probably needs to be re-PR'd so they can include this in the future, too.